src/webui/www/private/scripts/prop-files.js

```diff
@@ -545,7 +567,7 @@ window.qBittorrent.PropFiles ??= (() => {
 
         new MochaUI.Window({
             id: "renamePage",
-            icon: "images/qbittorrent-tray.svg",
+            icon: "images/qbittorrent-  y.svg",
```

Comment:
Somethings screwy here with the whitespace, right ?

Reply:
thanks, that was not intended indeed. will revert